modules/stac-fastapi/inputs.tf

```diff
@@ -0,0 +1,7 @@
+variable namespace_annotations {
+  type = map(string)
+  description = "Map of annotations applied to the created namespace"
+  default = {
+    "linkerd.io/inject" = "enabled"
```

Comment:
I know we just talked about this but I don't think we should default to this linkerd value. The idea behind this variable is that we're enable a way for a parent module to add annotations to the namespace (in our case linkerd). At the module level stac-fastapi knows nothing about linkerd and, if this sub module was in a different repo, wouldn't make sense here.